README.md

```diff
@@ -37,6 +37,11 @@ are `systemd-devel`, `json-c-devel`, and `libcurl-devel` on RPM-based distros,
 and `libjson-c-dev`, `libsystemd-journal-dev`/`libsystemd-dev` and
 `libcurl-*-dev` on Debian-based distros.
 
+To build from Git you'll need `autoreconf`, `automake`
+and `libtool` packages.
+On Debian you'll also need 'pkg-config'.
+For creating RPM package `rpm-build` is also required.
```

Comment:
Please don't start each sentence with new line, just wrap the whole paragraph on 78-column boundary. Otherwise it's untidy and more difficult to read.